self-hosted/install/installation-docker.md

```diff
@@ -1,83 +1,82 @@
 ---
-title: Install TimescaleDB from Docker container
+title: Install TimescaleDB on Docker
 excerpt: Install self-hosted TimescaleDB from a pre-built Docker container
 products: [self_hosted]
 keywords: [installation, self-hosted, Docker]
 ---
 
 import WhereTo from "versionContent/_partials/_where-to-next.mdx";
 import Skip from "versionContent/_partials/_selfhosted_cta.mdx";
+import SelfHostedDocker from "versionContent/_partials/_install-self-hosted-docker-based.mdx";
+import AddTimescaleDBToDB from "versionContent/_partials/_add-timescaledb-to-a-database.mdx";
 
-# Install TimescaleDB from a pre-built container
+# Install TimescaleDB from a Docker container
 
-You can install a TimescaleDB instance on any local system, from a pre-built
-container. 
+TimescaleDB is a [PostgreSQL extension](https://www.postgresql.org/docs/current/external-extensions.html) for
+time series and demanding workloads that ingest and query high volumes of data. You can install a TimescaleDB 
+instance on any local system, from a pre-built container. 
 
 < Skip/>
 
+This section shows you how to:
 
-This is the simplest method to self-host TimescaleDB, and it means you
-always have access to the latest version without worrying about local
-dependencies. You can access the Docker image directly from locally installed
-PostgreSQL client tools such as `psql`.
-
+* [Install and configure TimescaleDB on PostgreSQL](#install-and-configure-timescaledb-on-postgresql) - set up
+  a self-hosted PostgreSQL instance to efficiently run TimescaleDB1.
+* [Add the TimescaleDB extension to your database](#add-the-timescaledb-extension-to-your-database) - enable TimescaleDB features and
+  performance improvements on a database.
 
 <Highlight type="warning">
+
 If you have already installed PostgreSQL using a method other than the pre-built
-container provided here, you could encounter errors following these
-instructions. It is safest to remove any existing PostgreSQL installations
-before you begin. If you want to keep your current PostgreSQL installation, do
-not install TimescaleDB using this method.
-[Install from source](/self-hosted/latest/install/installation-source/)
-instead.
+container provided here, you may encounter errors
+following these install instructions. Best practice is to full remove any existing PostgreSQL
+installations before you begin.
```

Comment:
I'm not sure this advice is valid, or at least maybe it should be worded differently. Part of the benefit of the docker approach is being able to run many database instances on the same machine simultaneously. In my experience, the most frequent issue is trying to reuse the same default port (5432) for multiple instances. That fails. Maybe we should mention that here instead?

Reply:
Removed, added comment about the port when the container is started.